src/xunit.performance.api/GenericProfiler.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal static class GenericProfiler
```

Comment:
Do we even need this?  It looks like in the API could just call action.Invoke() directly.  There's no interface that we have to match.